public/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html ng-app="AlwaysDo">
+<head>
+  <link rel="icon" href="favicon.ico" />
+  <script src="angular/angular.js"></script>
+  <script src="angular-resource/angular-resource.js"></script>
+  <script src="js/app.js"></script>
+  <script src="js/mainCtrl.js"></script>
+  <link href='jura_font.css' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" type="text/css" href="styles.css">
+  <title></title>
+</head>
+
+<body ng-controller="MainCtrl as Main">
+<a class="header" href="index.html">
+  <h1> {{ Main.title }} </h1>
+</a>
+  <controls class="controls">
+    <form class="add_task" ng-submit="Main.addTask()">
+      <input type="text" ng-model="Main.task">
+      <button>Add Task!</button>
+    </form>
+    <filters>
+      View by:
+      <button ng-click="Main.filterByComplete()">Completed</button>
+      <button ng-click="Main.filterByActive()">Active</button>
+      <button ng-click="Main.removeFilter()">All</button>
```

Comment:
Might be able to alias this to `Main` anyway - and have this as `filterByComplete()`, `filterByActive()`, and `removeFilter()`